Source/Scene/Cesium3DTileset.js

```diff
@@ -2804,6 +2808,32 @@ Cesium3DTileset.prototype.destroy = function () {
   return destroyObject(this);
 };
 
+Cesium3DTileset.supportedExtensions = {
+  "3DTILES_metadata": true,
+  "3DTILES_implicit_tiling": true,
+  "3DTILES_content_gltf": true,
+  "3DTILES_multiple_contents": true,
+  "3DTILES_bounding_volume_S2": true,
+  "3DTILES_batch_table_hierarchy": true,
+  "3DTILES_draco_point_compression": true,
+};
+
+/**
+ * Checks to see if a given extension is supported by Cesium3DTileset. If
+ * the extension is not supported by Cesium3DTileset, it throws a RuntimeError.
+ *
+ * @param {Object} extensionsRequired The extensions we wish to check
+ *
+ * @private
+ */
+Cesium3DTileset.checkSupportedExtensions = function (extensionsRequired) {
+  for (var i = 0; i < extensionsRequired.length; i++) {
+    if (!Cesium3DTileset.supportedExtensions[extensionsRequired[i]]) {
+      throw new RuntimeError("Unsupported 3D Tiles Extension: " + extension);
```

Comment:
One last tiny thing, `extension` here is undefined in the error message, so it ends up throwing a ReferenceError. 

Fix this and I'll merge